blocks/image-placeholder/index.js

```diff
@@ -19,7 +25,12 @@ import MediaUpload from '../media-upload';
  */
 export default function ImagePlaceHolder( { className, icon, label, onSelectImage } ) {
 	const setImage = ( [ image ] ) => onSelectImage( image );
-	const dropFiles = ( files ) => mediaUpload( files, setImage );
+	const onFilesDrop = ( files ) => mediaUpload( files, setImage );
+	const onHTMLDrop = ( HTML ) => setImage( map(
+		rawHandler( { HTML, mode: 'BLOCKS' } )
+			.filter( ( { name } ) => name === 'core/image' ),
```

Comment:
This will take the first image if multiple images found. Is this the desired behavior?